monai/data/box_utils.py

```diff
@@ -0,0 +1,291 @@
+    """
+    This function returns BoxMode object from giving mode according to BOXMODE_MAPPING
+    Args:
+        mode: source box mode. If mode is not given, this func will assume mode is StandardMode
+        It can be:
+            #. str: choose from monai.utils.enums.BoundingBoxMode, including
+                "xyxy": boxes has format [xmin, ymin, xmax, ymax]
+                "xyzxyz": boxes has format [xmin, ymin, zmin, xmax, ymax, zmax]
+                "xxyy": boxes has format [xmin, xmax, ymin, ymax]
+                "xxyyzz": boxes has format [xmin, xmax, ymin, ymax, zmin, zmax]
+                "xyxyzz": boxes has format [xmin, ymin, xmax, ymax, zmin, zmax]
+                "xywh": boxes has format [xmin, ymin, xsize, ysize]
+                "xyzwhd": boxes has format [xmin, ymin, zmin, xsize, ysize, zsize]
+                "ccwh": boxes has format [xcenter, ycenter, xsize, ysize]
+                "cccwhd": boxes has format [xcenter, ycenter, zcenter, xsize, ysize, zsize]
+            #. BoxMode class: choose from
+                CornerCornerModeTypeA: equivalent to "xyxy" or "xyzxyz"
+                CornerCornerModeTypeB: equivalent to "xxyy" or "xxyyzz"
+                CornerCornerModeTypeC: equivalent to "xyxy" or "xyxyzz"
+                CornerSizeMode: equivalent to "xywh" or "xyzwhd"
+                CenterSizeMode: equivalent to "ccwh" or "cccwhd"
+            #. BoxMode instance
+            #. None: will assume mode is StandardMode
+    Returns:
+        BoxMode object
+
+    Example:
+        mode = "xyzxyz"
+        get_boxmode(mode) will return CornerCornerModeTypeA()
+    """
+    if isinstance(mode, BoxMode):
+        return mode
+
+    boxmode: Type[BoxMode]
+    if inspect.isclass(mode) and issubclass(mode, BoxMode):
+        boxmode = mode
+    elif isinstance(mode, str):
+        for m in SUPPORTED_MODES:
+            for n in SUPPORTED_SPATIAL_DIMS:
+                if m.get_name(n) == mode:
+                    boxmode = m
+    elif mode is None:
+        boxmode = StandardMode
+    else:
+        raise ValueError(f"Unsupported box mode: {mode}.")
+    return boxmode(*args, **kwargs)
+
+
+def check_corners(corners: Sequence) -> bool:
+    """
+    check the validity for the given box corners
+    Args:
+        corners: corners of a box, 4-element or 6-element tuple, each element is a Nx1 torch tensor
+        (xmin, ymin, xmax, ymax) or (xmin, ymin, zmin, xmax, ymax, zmax)
+    Returns:
+        bool, whether the box is valid
+    Example:
+        corners = (torch.ones(10,1), torch.ones(10,1), torch.ones(10,1), torch.ones(10,1))
+        check_corner(corners) will return True
+    """
+    spatial_dims = get_dimension(corners=corners)
+    box_error = corners[spatial_dims] < corners[0]
+    for axis in range(1, spatial_dims):
+        box_error = box_error | (corners[spatial_dims + axis] < corners[axis])
+    if box_error.sum() > 0:
+        return False
+    else:
```

Comment:
This `else` will fail CI format test, maybe we can simplify it to:
```py
spatial_dims = get_dimension(corners=corners)
for axis in range(spatial_dims):
    if corners[spatial_dims + axis] < corners[axis]
        return False
return True
```
What do you think?

Thanks.